src/read.c

```diff
@@ -1728,7 +1728,7 @@ static avifBool avifParseAuxiliaryTypeProperty(avifProperty * prop, const uint8_
     return AVIF_TRUE;
 }
 
-static avifBool avifParseColourInformationBox(avifProperty * prop, const uint8_t * raw, size_t rawLen, avifDiagnostics * diag)
+static avifBool avifParseColourInformationBox(avifProperty * prop, const uint8_t * raw, uint64_t rawOffset, size_t rawLen, avifDiagnostics * diag)
```

Comment:
Nit: Since a {pointer, length} pair is a common construct, I suggest not inserting an offset in the middle. How about listing the three parameters in the order of raw, rawLen, rawOffset or rawOffset, raw, rawLen?

This change would need to be made to all the functions that take these three parameters.